mail-filter/mimedefang/files/mimedefang.init

```diff
@@ -0,0 +1,104 @@
+#!/sbin/openrc-run
+#
+# Generic start/stop script for MIMEDefang.  Should work on most
+# flavors of UNIX.
+
+RETVAL=0
+prog='mimedefang'
```

Comment:
This seems entirely meaningless, especially that it's hacked to construct two different names below.

Reply:
I don't quite understand why you say it's meaningless, but please see new commit.

Reply:
I'm saying putting `mimedefang` inline instead of `${prog}` would be more readable. There's no reason for prog to ever have a different value, is it?